libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -65,6 +65,35 @@
 #include <stdio.h>
 #include <string.h>
 
+/**
+ * @brief Represents string to be loggedwhen mbed TLS returned error
```

Comment:
nit: logged when typo

Reply:
Fixed

Reply:
Oops, I forgot to push the change. I will make the change in a follow-up PR.